### PR TITLE
Add varnish plan

### PR DESCRIPTION
**Pull request by kevindickerson**
Signed-off-by: Kevin J. Dickerson <kevin.dickerson@loom.technology>

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @adamhjk to be a potential reviewer

File: plans/varnish/plan.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  PCRE_LIBS="-L$(pkg_path_for core/pcre)/lib -lpcre" \
+  PCRE_CFLAGS="-I$(pkg_path_for core/pcre)/include" \
+  ./configure --prefix=$pkg_prefix --sbindir=${pkg_prefix}/bin
+  make
+}
+
+do_install() {
+  do_default_install
+  mkdir -p $pkg_svc_var_path/habitat_varnish
+  chmod 777 $pkg_svc_var_path/habitat_varnish
```
Comment (jtimberman) [inline plans/varnish/plan.sh]: Probably don't want world write. `755` perhaps?

**Comment by kevindickerson** [comment]
Feel free to comment about the config.. it's bare-bones right now

File: plans/varnish/plan.sh (hunk 1 above)
Comment (kevindickerson) [inline plans/varnish/plan.sh]: So I was wondering what the right thing to do with this is... Is umask better than chmod 755? chmod? Not quite sure about how to configure the hab environment's umask, but I figured chmod might be a good start?

File: plans/varnish/plan.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
+pkg_origin=loom
```
Comment (kevindickerson) [inline plans/varnish/plan.sh]: d'oh

File: plans/varnish/plan.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  PCRE_LIBS="-L$(pkg_path_for core/pcre)/lib -lpcre" \
+  PCRE_CFLAGS="-I$(pkg_path_for core/pcre)/include" \
+  ./configure --prefix=$pkg_prefix --sbindir=${pkg_prefix}/bin
+  make
+}
+
+do_install() {
+  do_default_install
+  mkdir -p $pkg_svc_var_path/habitat_varnish
+  chmod 755 $pkg_svc_var_path/habitat_varnish
```
Comment (kevindickerson) [inline plans/varnish/plan.sh]: Insufficient permissions with 755... fixing

**Comment by smith** [comment]
Looks like this fork is no longer available to review. Closing.

**Pull request closed without merging**